decidim-core/app/models/decidim/user.rb

```diff
@@ -51,6 +51,10 @@ def role?(role)
       roles.include?(role.to_s)
     end
 
+    def verified?
```

Comment:
Docs. And I'm not sure about the naming. Verification is not the same as authorization. We could add some kind of verification (via a third service that scans an ID) but this doesn't mean the user has been authorised with the census or any other authorization.